Cargo.toml

```diff
@@ -25,7 +25,7 @@ boa_runtime = { version = "0.19" }
 bytes = { version = "1" }
 clap = { version = "4.5" }
 common-builder = { path = "./rust/common-builder" }
-common-javascript-interpreter = { path = "./rust/common-javasript-interpreter" }
+common-javascript-interpreter = { path = "./rust/common-javascript-interpreter" }
```

Comment:
This could probably just be removed, as it's not a "cargo dependency" of any existing crates, though while `common-runtime` *depends* on it, due to the different build target that dependency is expressed via the `build.rs` script